src/Domain/Python/ElementId.cpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <cstddef>
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+
+#include "Domain/ElementId.hpp"
+#include "Domain/SegmentId.hpp"
+#include "Utilities/GetOutput.hpp"
+
+namespace py = pybind11;
+
+namespace domain {
+namespace py_bindings {
+
+namespace detail {
+template <size_t Dim>
+void bind_element_id_impl(py::module& m) {  // NOLINT
```

Comment:
These bindings seem to omit a number of functions in the public interface of the `ElementId` and `SegmentId` classes. I'm fine with the partially created bindings, but it would probably be helpful to include a code comment explaining that there are a number of absent functions that can be added later.